### PR TITLE
Remove antiGravityMode and change iTermAcceleratorGain to antiGravityGain

### PR DESCRIPTION
**Pull request by haslinghuis**
Should be combined with ~~https://github.com/betaflight/betaflight/pull/11656~~ https://github.com/betaflight/betaflight/pull/11679

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by limonspb — DISMISSED** [review]

**Review by KarateBrot — DISMISSED** [review]

**Review by blckmn — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Comment by haslinghuis** [comment]
Needs rework now https://github.com/betaflight/betaflight/pull/11656 has been superseded by https://github.com/betaflight/betaflight/pull/11679

**Comment by ctzsnooze** [comment]
This seems really good to me.  For 4.4 it definitely shows the correct value.
I would approve if I had permission to approve :-)

**Review by ctzsnooze — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
